src/Drupal/Generators/entity/bundle_class.php.twig

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Drupal\{{ machine_name }}\Entity\Bundle\{{ entity_type_id }};
+{% if not use_base_class %}
+
+use {{ parent_class }};
+{% endif %}
+
+/**
+ * A bundle class for {{ entity_type_id }} entities.
+ */
+class {{ bundle_class }} extends {{ parent_class|split('\\')|last }} {
```

Comment:
IMHO, we should encourage bundle classes to extend bundle interfaces. A `Page` bundle class could implement ` `PageInterface`. This is very useful, as a bundle class can implement multiple interfaces:

```php

interface PageInterface {
  public function getContent();
}

interface ArchivableInterface {
  public function getArchivedDate();
}

class Page implements PageInterface, ArchivableInterface {
}
```

also this allows to do some more semantically checks:

```php
if ($node instanceof PageInterface) {...}
```